OutSystems/outsystems-datagrid-reactive
# RGRIDT-700: OnCellValueChange class from ColumnEvents should extend AbstractColumnEvent

This PR is for RGRIDT-700: OnCellValueChange class from ColumnEvents should extend AbstractColumnEvent

### What was happening
* OnCellValueChange class from ColumnEvents should extend AbstractColumnEvent. Right now it is extending InternalEvents.AbstractEvent which is not the right class to extend from when compared with the UML of the grid.

### What was done
* Changed the OnCellValueChange class from ColumnEvents to extend from AbstractColumnEvent.
* Changed the trigger method from AbstractColumnEvent in order to facilitate invocations of the trigger method with different arguments. 
* 1. ActionColumnClick - trigger(gridID, columnID, line)
* 2. OnCellValueChange - trigger(gridID, collumnID, rowNumber, oldValue, newValue)

### Test Steps
1.Click on any cell from Action Column
**Expected**: a feedback message should appear with the id of the line that was clicked.

1.Edit the first cell from Text Column (Product Name) with the value "this is going to trigger an error"
**Expected**: a red border and a red mark should appear on the cell. On hovering the cell with the mouse, a tooltip with an error should appear as well.

### Checklist
